core/src/main/java/org/apache/calcite/runtime/CalciteResource.java

```diff
@@ -948,8 +948,8 @@ ExInst<CalciteException> illegalArgumentForTableFunctionCall(String a0,
   @BaseMessage("Illegal arguments: The length of the keys array {0,number,#} is not equal to the length of the values array {1,number,#} in MAP_FROM_ARRAYS function")
   ExInst<CalciteException> illegalArgumentsInMapFromArraysFunc(int arg0, int arg1);
 
-  @BaseMessage("Trim error: trim character must be exactly 1 character")
```

Comment:
this should read "the length of the string describing the trimmed character should be 1"

Reply:
I've noticed that the former doc implies changing it to `Invalid argument...`.Since it is a strict rule in SQL standard, what about "Invalid argument ''{0}'' : the length of the string describing the trimmed character must be 1".

Reply:
> this should read "the length of the string describing the trimmed character should be 1"

Done.